src/App.tsx

```diff
@@ -14,210 +34,132 @@ export const App: React.FC = () => {
     }
   }, []);
 
+  const getUserTodos = useCallback(async () => {
+    if (user) {
+      try {
+        setIsError(false);
+
+        const todosFromServer = await getTodos(user.id);
+
+        setTodos(todosFromServer);
+      } catch (error) {
+        setErrorText('Unable to connect to server');
+        setIsError(true);
+      }
+    }
+  }, []);
+
+  const getVisibleGoods = useCallback((allTodos: Todo[]) => {
+    return allTodos.filter(todo => {
```

Comment:
you can delete 'return' here
```suggestion
  const getVisibleGoods = useCallback((allTodos: Todo[]) => (
    allTodos.filter(todo => {
```

Reply:
nope, i can`t do it without return here, cause i need to return new array